docs/reference-implementation/Developer-Guide.md

```diff
@@ -0,0 +1,58 @@
+# Developer Guide
```

Comment:
since the APIM stuff might be a while until we want to surface it here, how about splitting the dev guide in a separate PR? That is definitely very valuable already, and we also might want to port it to the Online reference implementation later